pom.xml

```diff
@@ -76,15 +76,28 @@ under the License.
         <java.version>1.8</java.version>
         <spotless-maven-plugin.version>1.20.0</spotless-maven-plugin.version>
         <auto-service.version>1.0-rc6</auto-service.version>
-        <protobuf.version>3.7.1</protobuf.version>
-        <unixsocket.version>2.3.2</unixsocket.version>
-        <protoc-jar-maven-plugin.version>3.11.1</protoc-jar-maven-plugin.version>
-        <flink.version>1.15.2</flink.version>
+        <protobuf.version>3.23.2</protobuf.version>
+        <unixsocket.version>2.6.2</unixsocket.version>
+        <protoc-jar-maven-plugin.version>3.11.4</protoc-jar-maven-plugin.version>
+        <flink.version>1.17.1</flink.version>
         <scala.binary.version>2.12</scala.binary.version>
         <scala.version>2.12.7</scala.version>
         <lz4-java.version>1.8.0</lz4-java.version>
-        <flink-shaded-jackson.version>2.12.4-15.0</flink-shaded-jackson.version>
+        <flink-shaded-jackson.version>2.14.2-17.0</flink-shaded-jackson.version>
         <slf4j-log4j12.version>1.7.32</slf4j-log4j12.version>
+        <flink-connector-kinesis.version>4.1.0-1.17</flink-connector-kinesis.version>
+        <flink-connector-aws-kinesis-streams.version>4.1.0-1.17</flink-connector-aws-kinesis-streams.version>
+        <okhttp.version>3.14.6</okhttp.version>
+        <flink-shaded-netty.version>4.1.82.Final-16.1</flink-shaded-netty.version>
+        <junit.version>4.12</junit.version>
+        <hamcrest-all.version>1.3</hamcrest-all.version>
+        <kryo.version>2.24.0</kryo.version>
+        <jackson-databind.version>2.13.2.2</jackson-databind.version>
+        <flink-shaded-netty.version>4.1.82.Final-16.1</flink-shaded-netty.version>
+        <flink-shaded-force-shading.version>16.1</flink-shaded-force-shading.version>
+        <commons-codec.version>1.15</commons-codec.version>
+        <commons-logging.version>1.2</commons-logging.version>
+        <slf4j-api.version>1.7.36</slf4j-api.version>
```

Comment:
It doesn't appear as we re-use these variables anywhere else, should we just leave them as they were?